python/ray/ml/BUILD

```
@@ -129,6 +129,14 @@ py_test(
     deps = [":ml_lib"]
 )
 
+py_test(
+    name = "test_horovod_trainer",
+    size = "large",
```

does this need to be large?

```suggestion
    size = "medium",
```

---

yes